cmd/config.go

```diff
@@ -188,6 +188,13 @@ type Config struct {
 		BadResultsOnly      bool
 		CheckPeriod         ConfigDuration
 	}
+
+	ExpiredAuthzPurger struct {
```

Comment:
Let's put this config JSON in expired-authz-purger itself. But let's embed it in an anonymous top-level struct so that the config file has a field saying what it's for. E.g.:

```
struct eapConfig {
  ExpiredAuthzPurger struct {
    ...
  }
}
```